include/dlpack/dlpack.h

```diff
@@ -81,8 +81,8 @@ typedef enum {
   kDLUInt = 1U,
   kDLFloat = 2U,
   kDLBfloat = 4U,
-  kDLComplex = 8U,    // C/C++/Python layout (compact struct per complex number)
-  kDLComplexS = 16U,  // alternative layout (a struct with two pointers to real & imag arrays)
+  kDLComplex = 5U,    // C/C++/Python layout (compact struct per complex number)
+  kDLComplexS = 6U,  // alternative layout (a struct with two pointers to real & imag arrays)
```

Comment:
Given that the Struct of Array is less conventionally defined(we need to define how the layout is, right now seems only well defined for compact array), let us first start with Complex(which is easier to agree on) then discuss the SOA case later.

Reply:
Sounds good, @tqchen. This is also what I suggested in #50. I removed `kDLComplexS` in 40beada. PTAL.